httpclient/src/test/kotlin/tbdex/sdk/httpclient/UtilsTest.kt

```diff
@@ -0,0 +1,62 @@
+package tbdex.sdk.httpclient
+
+import com.nimbusds.jwt.SignedJWT
+import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.methods.dht.DidDht
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertTrue
+
+class UtilsTest {
+
+  @Test
+  fun `generateRequestToken() generates a JWT`() {
+    val did = DidDht.create(InMemoryKeyManager())
+    val pfiDid = "did:ion:123"
+
+    val token = generateRequestToken(did, pfiDid)
+    assertNotNull(token)
+  }
+
+  @Test
+  fun `generateRequestToken() generates JWT with all required fields`() {
+    val did = DidDht.create(InMemoryKeyManager())
+    val pfiDid = "did:ion:123"
+
+    val token = generateRequestToken(did, pfiDid)
+    val claimsSet = SignedJWT.parse(token).jwtClaimsSet
+
+    val requiredKeys = listOf("aud", "iss", "exp", "jti", "iat")
```

Comment:
this list is duplicated between test and impl, should extract out to a const in the utils file